### DISCUSSION TITLE
Cannot use WebDav Digest Auth

### DISCUSSION THREAD

**Discussion by rjhub1010** [floccusaddon/floccus#1541] [General]
I am checking out Floccus and like it.  But I cannot connect to my WebDav server unless I disable Digest Authentication.  I do SSL and have a cert but am not inclined to remove digest from my auth.  My other self-hosted tools work with it and hoping to add this if it can be resolved.

**Comment by marcelklehr** [comment]
Sorry, floccus only supports Basic Auth
